charts/incubator/duplicacy/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
-appVersion: "latest"
+appVersion: "release-1.7.2"
```

Comment:
```suggestion
appVersion: "1.7.2"
```

Reply:
will that work even though its tagged release-1.7.2?

Reply:
done